docs/doc/30-reference/20-functions/11-conditional-functions/multi_if.md

```diff
@@ -0,0 +1,49 @@
+---
+title: MULTI_IF
```

Comment:
Shall we add a documentation for `CASE WHEN` as well?

Reply:
It could be done after we enable new planner.

Reply:
We can add `CASE WHEN` to the doc with the settings `SET enable_planner_v2 = 1;`, like https://databend.rs/doc/reference/functions/conditional-functions/nullif

cc @soyeric128

Reply:
Hi @BohuTANG and @sundy-li 
Pls review this PR: https://github.com/datafuselabs/databend/pull/6133